pl-PL/Happy Birthday/Happy Birthday - notes.md

```diff
@@ -1,30 +1,31 @@
 ---
-title: Happy Birthday — Notes for Club Leaders
-language: en-GB
+title: Happy Birthday — Informacje dla prowadzącego
+language: pl-PL
 embeds: "*.png"
 materials: [""]
 ...
 
-#Introduction:
-In this project, children will be introduced to HTML tags & CSS properties. They will also edit tags and properties to make their own customised greeting card.
+# Wstęp
+Wykonując ten projekt dzieci poznają tagi HTML i właściwości CSS. Podczas zajęć będą także zmieniać tagi i właściwości, aby stworzyć swoją własną kartkę okolicznościową.
```

Comment:
może lepiej: "zapoznają się z tagami" - by podkreślić, że to melodia przyszłości?